Google.Api.Generator/Generation/MetadataGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Google.Gapic.Metadata;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+
+namespace Google.Api.Generator.Generation
+{
+    internal static class MetadataGenerator
+    {
+        private const string GapicMetadataSchemaVersion = "1.0";
+        private const string GapicMetadataFileDescription = "This file maps proto services/RPCs to the corresponding library clients/methods";
+        private const string GapicMetadataLanguage = "csharp";
+
+        private const string TransportKeyGrpc = "grpc";

```

Comment:
How will this fit in with REGAPIC work?

Reply:
it will have to be updated in the REGAPIC PRs